### ISSUE TITLE
Sharing Cart plugin issue - content run out of the screen to right

### ISSUE THREAD

**Issue by jp-edusampo** [donhinkelman/moodle-block_sharing_cart#161]
Hello,

I sended this issue to Moodle forum and already got solution there. But I also got request to send this issue here, so here it is.

-------------
Hello!

We recently updated our Moodle from 4.0.4. to 4.0.6. After that we got little bug with the Sharing Cart -plugin, Images should show the issue:

![image](https://user-images.githubusercontent.com/127884851/225093122-d35ced25-d27c-4922-a068-5ee46dc4fa51.png)

Sharing cart version is 4.0, release 2 and we use Adaptable -theme.

-----------------
Solution by Stefan Kohler:

we had the same problem with the combination of sharing-cart and Adaptable.
As solution I put the following 

[CSS](https://moodle.org/mod/glossary/showentry.php?eid=13&displayformat=dictionary)-Code into the customcss-field of the Theme:

```
.block_sharing_cart ul.tree.list {display: contents;}
.block_sharing_cart ul.tree.list > li {display: inherit;}
.block_sharing_cart ul.tree.list > li .toggle-wrapper {width: auto;}
.block_sharing_cart ul.tree.list > li span.commands {margin-left: 5px;}
.block_sharing_cart ul.tree.list > li ul.list {display: block;}
.block_sharing_cart ul.tree.list > li ul.list > li {display: block;}
.block_sharing_cart ul.tree.list > li ul.list .instancename {width: auto;}
.block_sharing_cart ul.tree li > div .instancename {width: auto;}
.block_sharing_cart ul.tree.list > li ul.list::before,
.block_sharing_cart ul.tree.list > li ul.list li::before {content: none; outline: none;}
```

---------------

**Comment by donhinkelman** [comment]
Hello, edusampo. Thank you for your fix. Praxis has incorporated it into the newest version. Could you confirm that Sharing Cart 4.0, release 5 is working with no problems?

**Comment by thepurpleblob** [comment]
I'm not sure I understand your numbering. We have SC 4.1 Release 1 (build 2023032300) and the problem persists. The suggested CSS fix (above) sorts it. 

Should this version have the fix? It's the latest available for 4.1

**Comment by wichmann** [comment]
Hello,

just wanted to give you a feedback: I am using Moodle 4.1.4 (Build: 20230612) with the sharing cart plugin version 4.1, release 12023032300. In both themes "Boost" and "Boost Union" the problem still exists (see screenshots). The CSS solution above seems to eliminate the error. Can you help me? Thanks!

Best regards,
  Christian Wichmann

![Screenshot_20230727_122205](https://github.com/donhinkelman/moodle-block_sharing_cart/assets/4911095/2648e8ed-7cc7-4f53-a19b-23dded0dce2b)
![Screenshot_20230727_122220](https://github.com/donhinkelman/moodle-block_sharing_cart/assets/4911095/8e67eca0-6f57-4dd3-9b84-9fa615d57562)

**Comment by AraPraxis** [comment]
Hello @jp-edusampo 

I do believe your issue can be fixed by pulling a newer release of sharing cart.
We've had a similar issue (https://github.com/donhinkelman/moodle-block_sharing_cart/issues/165) and it should've been fixed in Moodle 4.3 Release 1

If you have any further problems or questions, then feel free to open the issue again or create a new one.

Kind regards,
Anders Rasmussen

Moxis Team
